ex03.exs

```diff
@@ -53,6 +53,42 @@ defmodule Ex03 do
   def pmap(collection, process_count, function) do
     # your code goes here
     # I'm hoping to see a simple pipeline as the body of this function...
+    range_to_enumerable(collection)
+    |> splitmap(process_count)
+    |> spawnchunks(function)
+    |> receivechunks
+    |> flatten_result
+  end
+
+  def range_to_enumerable(range) do
+    Enum.map(range, fn x -> x end)
+  end
```

Comment:
Range is already enumerable. In fact, this function makes that assumption, as `map` wouldn't work otherwise...